### PR TITLE
New event page for instructors, creates EventInfo and SectionEvent

### PR DESCRIPTION
**Pull request by seveibar**

### CODE REVIEW

File: client/components/eventform/eventform.controller.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    $scope.submitEventAssignment = (form)=>{
+        $scope.submitted = true;
+        getSectionIds().forEach((sectionId)=>{
+          var sectionEvent = {
+            section: sectionId,
+            info: $scope.eventInfo._id,
+            author: Auth.getCurrentUser()._id,
+            additionalNotes: $scope.event.additionalNotes
+          };
+          SectionEvent.create(sectionEvent).$promise
```
Comment (jowlee) [inline client/components/eventform/eventform.controller.js:102]: On server side add creation date

File: client/components/eventform/eventform.controller.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      $scope.courses = _.values(coursesObj);
+    });
+
+    $scope.useEvent = (event)=>{
+      eventInfoId = event._id;
+      $scope.selectingEvent = false;
+      $scope.eventInfo = event;
+    };
+
+    $scope.createEventInfo = (form)=>{
```
Comment (jowlee) [inline client/components/eventform/eventform.controller.js:33]: Add times for when the event is happening to the form, creation date and author

Comment (seveibar) [inline client/components/eventform/eventform.controller.js:33]: Neither author nor creation date need to be added (the first is added in the controller, the second is added by mongoose serverside). The time does need to be added, but we may want a robust time selection system- so im recommending putting it in the PR with location

File: client/components/eventform/eventform.controller.js (hunk 1 above)
Comment (seveibar) [inline client/components/eventform/eventform.controller.js:102]: this should be in the model- if not it belongs in the model (essentially initialValue: currentTime)

**Comment by seveibar** [comment]
Created #122 to address the form fields that are not in this PR

**Comment by jowlee** [comment]
Great work

**Comment by seveibar** [comment]
The model still needs to be fixed- I'll do another PR for it.

I just looked at [the model](https://github.com/rcos/venue/blob/dev/server/api/sectionevent/sectionevent.model.js) and creation date does not have a default

**Pull request merged**